File: app/src/main/java/vn/loitp/app/activity/database/readsqliteasset/ReadSqliteAssetActivity.java
package vn.loitp.app.activity.database.readsqliteasset;

import android.os.Bundle;
import android.widget.TextView;

import com.core.base.BaseFontActivity;
import com.core.utilities.LLog;
import com.core.utilities.LUIUtil;

import java.util.ArrayList;
import java.util.List;

import loitp.basemaster.R;

public class ReadSqliteAssetActivity extends BaseFontActivity {
    private VocabularyManager vocabularyManager;
    private TextView tv;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        vocabularyManager = new VocabularyManager(getActivity());
        try {
            vocabularyManager.createDatabase();
            LLog.d(getTAG(), "init dtb success");
        } catch (Exception e) {
            LLog.d(getTAG(), "init dtb failed: " + e.toString());
        }
        List<Vocabulary> vocabularyList = new ArrayList<>();
        vocabularyList.addAll(vocabularyManager.getAllVocabulary());

        LLog.d(getTAG(), "size: " + vocabularyList.size());

        tv = findViewById(R.id.tv);
        LUIUtil.printBeautyJson(vocabularyList.get(0), tv);
    }

    @Override
    protected boolean setFullScreen() {
        return false;
    }

    @Override
    protected String setTag() {
        return getClass().getSimpleName();
    }

    @Override
    protected int setLayoutResourceId() {
        return R.layout.activity_read_sqlite_asset;
    }

}

File: app/src/main/java/vn/loitp/app/activity/database/readsqliteasset/ReadSqliteAssetActivity.kt
package vn.loitp.app.activity.database.readsqliteasset

import android.os.Bundle
import com.loitpcore.annotation.IsFullScreen
import com.loitpcore.annotation.LogTag
import com.loitpcore.core.base.BaseFontActivity
import com.loitpcore.core.utilities.LUIUtil
import kotlinx.android.synthetic.main.activity_read_sqlite_asset.*
import vn.loitp.app.R

@LogTag("ReadSqliteAssetActivity")
@IsFullScreen(false)
class ReadSqliteAssetActivity : BaseFontActivity() {

    override fun setLayoutResourceId(): Int {
        return R.layout.activity_read_sqlite_asset
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setupViews()
    }

    private fun setupViews() {
        val vocabularyManager = VocabularyManager(this)
        try {
            vocabularyManager.createDatabase()
            logD("init dtb success")
            showLongDebug("init dtb success")
        } catch (e: Exception) {
            logE("init dtb failed: $e")
            showDialogError("init dtb failed: $e")
            return
        }

        val vocabularyList = ArrayList(vocabularyManager.allVocabulary)
        logD("size: " + vocabularyList.size)

        try {
            LUIUtil.printBeautyJson(o = vocabularyList[0], textView = textView)
        } catch (e: Exception) {
            textView.text = "$e"
        }
    }
}
